packages/components/src/table/test/index.js

```diff
@@ -93,5 +116,7 @@ describe( 'TableCard', () => {
 				screen.queryByText( row[ 0 ].display )
 			).toBeInTheDocument();
 		}
+
+		expect( container ).toMatchSnapshot();
```

Comment:
We don't need a snapshot test here, as structural testing has already been done in the first few tests. Also, we already have a behavioral test written at line 114.

Reply:
Addressed here 667be04a9eddeefb1ffa6e64b36c7fd8f38e1913